addons/account/models/chart_template.py

```diff
@@ -437,7 +438,7 @@ def deref(values, model):
                         values[fname] = self.ref(value).id if value not in ('', 'False', 'None') else False
                     except ValueError as e:
                         _logger.warning("Failed when trying to recover %s for field=%s", value, field)
-                        raise e
+                        failed_fields.append(fname)
```

Comment:
Can't we just `del values[fname]` here?

Reply:
Tried that first as well, but Python is being too smart, with `fields` actually looping over `values` and it doesn't allow changing the size during iteration:

```
...
  File "/Users/dylankiss/src/odoo-multiverse/master/odoo/addons/account/models/chart_template.py", line 521, in _load_data
    'values': deref(record, self.env[model]),
  File "/Users/dylankiss/src/odoo-multiverse/master/odoo/addons/account/models/chart_template.py", line 430, in deref
    for field, fname, value in fields:
  File "/Users/dylankiss/src/odoo-multiverse/master/odoo/addons/account/models/chart_template.py", line 428, in <genexpr>
    fields = ((model._fields[k], k, v) for k, v in values.items() if k in model._fields)
RuntimeError: dictionary changed size during iteration
...
```